crates/db-boot/src/x86_64/mptable.rs

```diff
@@ -0,0 +1,523 @@
+// Copyright 2022 Alibaba Cloud. All Rights Reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
You should also include the original Google BSD license file.

Reply:
Google license file from crosvm has been added as THIRD-PARTY file.
Besides, Apache License is added under db-boot.
Will regulate license display in other pull requests.